app/components/Sidebar/components/SidebarContext.ts

```diff
@@ -1,9 +1,20 @@
 import * as React from "react";
 
-export type SidebarContextType = "collections" | "starred" | string | undefined;
+export type SidebarContextType =
+  | "collections"
+  | "shared"
+  | `group-${string}`
```

Comment:
`group-${string}`

I had no idea you can do this… are you sure?

Reply:
Yes, it works!

Reply:
mind blown